scripts/get_tool_dl_url.py

```diff
@@ -39,7 +53,7 @@ class StandardTool:
             ARCH_PATTERN = re.compile("all|arm64")
         else:
             sys.exit(f"Unsupported architecture {ARCH}")
-    elif OS == "Linux":
+    elif OS == "Linux" and ARCH in X86_64_ARCH_NAMES:
```

Comment:
Oopsie. Want y'all to see that this passed CI before I fix this.

Reply:
I don't think Linux / Arm combo will be used here, so it's ok to merge.